caput/pipeline/runner/_scheduler/__init__.py

```diff
@@ -87,24 +91,23 @@ def template_queue(templatefile, var, overwrite, email=None, mailtype=None):
 
 
 def queue(
-    configfile,
-    submit=False,
-    lint=True,
-    profile=False,
-    profiler="cProfiler",
-    psutil=False,
-    overwrite="never",
-    email=None,
-    mailtype=None,
+    configfile: str,
+    submit: bool = False,
+    lint: bool = True,
+    profile: bool = False,
+    profiler: str | None = "cProfiler",
+    psutil: bool = False,
+    overwrite: str = "never",
+    email: str | None = None,
+    mailtype: str | None = None,
 ):
-    r"""Queue a pipeline on a cluster from the given CONFIGFILE.
+    """Queue a pipeline on a cluster from the given CONFIGFILE.
```

Comment:
```suggestion
    """Queue a pipeline on a cluster from the given `configfile`.
```
Also needs a parameters section.